### DISCUSSION TITLE
Welcome to vscode-cmake-tools Discussions!

### DISCUSSION THREAD

**Discussion by bobbrow** [microsoft/vscode-cmake-tools#2183] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by kepeter** [comment]
Hi,

After working with CMake on VS 2022, it is time to check VSCode... (the real reason that I need the project on Linux too :-))
After a lot of fail with the big project, I created a one-liner to make things simple...
The project compiles without any problem under VS, but fails because of missing standard header files under VSCode...

This is the error:
**fatal error C1083: Cannot open include file: 'iostream': No such file or directory**

The project is attached as zip file for examination...
[CMakeProject.zip](https://github.com/microsoft/vscode-cmake-tools/files/9078930/CMakeProject.zip)

**Reply by bobbrow** [reply]
We opened an issue for this and we'll respond on that thread.  You can go here to follow it: #2647
